examples/machine_learning/Classification_Bitter.py

```diff
@@ -0,0 +1,386 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Bitterness Classification With XGBoost
```

Comment:
We need to add a README file in `example/assets` and explain what the "bitter-sweet.csv` file is. The name is ambiguous again.

Reply:
Added